src/SME.SGP.Aplicacao/Commands/PendenciaProfessor/ExecutarVerificacaoPendenciaAvaliacaoProfessor/ExecutarVerificacaoPendenciaAvaliacaoProfessorCommandHandler.cs

```diff
@@ -74,11 +74,11 @@ private async Task<bool> ExistePendenciaProfessor(Turma turma, TurmaEComponenteD
 
         private async Task IncluirPendenciaProfessor(Turma turma, long componenteCurricularId, string professorRf, int bimestre, string componenteCurricularNome, long periodoEscolarId)
         {
-            var escolaUe = $"{turma.Ue.TipoEscola.ShortName()} {turma.Ue.Nome} (DRE - {turma.Ue.Dre.Abreviacao})";
+            var escolaUe = $"{turma.Ue.TipoEscola.ShortName()} {turma.Ue.Nome} ({turma.Ue.Dre.Abreviacao}) - Turma {turma.Nome}";
             var titulo = $"Ausência de avaliação no {bimestre}º bimestre {escolaUe}";
 
             var descricao = $"<i>O componente curricular {componenteCurricularNome} não possui nenhuma avaliação cadastrada no {bimestre}º bimestre - {escolaUe}</i>";
-            var instrucao = "Acesse a tela de Calendário Escolar e confira os eventos da sua UE.";
+            var instrucao = "<i style='color:#FF0000';>Acesse a tela de Calendário Escolar e confira os eventos da sua UE.</i>";
```

Comment:
remover cor vermelha
```suggestion
            var instrucao = "Acesse a tela de Calendário Escolar e confira os eventos da sua UE.";
```